src/component/playground/Playground/PlaygroundResultsTable/FeatureResultInfoPopoverCell/PlaygroundResultFeatureStrategyList/PlaygroundResultStrategyList/PlaygroundResultFeatureStrategyItem/PlaygroundResultStrategyExecution/PlaygroundResultStrategyExecutionParameters/PlaygroundResultStrategyExecutionParameters.tsx

```diff
@@ -0,0 +1,136 @@
+import {
+    parseParameterNumber,
+    parseParameterStrings,
+} from 'utils/parseParameter';
+import { Box, Chip } from '@mui/material';
+import PercentageCircle from 'component/common/PercentageCircle/PercentageCircle';
+import { PlaygroundConstraintItem } from '../PlaygroundConstraintItem/PlaygroundConstraintItem';
+import React from 'react';
+import { useStyles } from '../PlaygroundResultStrategyExecution.styles';
+import {
+    PlaygroundConstraintSchema,
+    PlaygroundRequestSchema,
+} from 'hooks/api/actions/usePlayground/playground.model';
+import { getMappedParam } from '../helepers';
+
+export interface PlaygroundResultStrategyExecutionParametersProps {
+    parameters: { [key: string]: string };
+    constraints: PlaygroundConstraintSchema[];
+    input?: PlaygroundRequestSchema;
+}
+
+export const PlaygroundResultStrategyExecutionParameters = ({
+    parameters,
+    constraints,
+    input,
+}: PlaygroundResultStrategyExecutionParametersProps) => {
+    const { classes: styles } = useStyles();
+    const renderParameters = () => {
+        return Object.keys(parameters).map(key => {
+            switch (key) {
+                case 'rollout':
+                case 'Rollout':
```

Comment:
Would it be better to upcase/downcase the key here and match on that instead (to remove the dupes)? Something like:

```suggestion
            switch (key.toUpperCase()) {
                case 'ROLLOUT':
```

Reply:
sure